TOPSPIN/Phase_Correction/4D.md

```diff
@@ -1,36 +1,286 @@
+1 SI SI*4
+```
+
+- These commands have the following effects:
+  - Leave the default `ABSF1` and `ABSF2` values in the `N` dimension.
+  - Select the `qpol` polynomial function since `qfil` suppresses water very aggressively and is not recommended for the `N-HN` spectrum.
+  - Set `BC_mod` to `no` in the `N` dimension due to its sensitivity to the polynomial.
+  - Set the `SSB` of the direct dimension to `2.2` for further resolution.
+  - Increase the `SI` size of the direct dimension by 2 times to enhance resolution.
+  - Increase the indirect dimension size by four times (zero filling). Ensure that **all SI values are powers of 2** (e.g., 1024, 2048, etc.).
+
+- Execute `xfb` followed by `abs1` and `abs2` for baseline correction. This will remove the irrelevant HN shift region,
+including the water line, while keeping only the ROI. It will also result in sharper and better-resolved peaks due to 
+zero filling and increased SSB.
+
+3. **Prepare for Manual Phase Correction in the F2 dimension:**
+  - Enter `.ph` for manual phase correction.
+  - Look for peaks that have an anti-phase component (signals of dispersion character) in the F2 dimension.
+  - Place the cursor between the centers of these two peaks. Ensure the cursor is along the line connecting them. 
+  - Right-click and select **"Add"**.
+
+![Add Peak](images/add_peak.png)
+
+   - Try to select peaks that cover the span of the whole spectrum; 4 or 5 peaks should suffice.
+
+![Added Peaks](images/added_peaks.png)
+
+4. **Enter Phase Correction Mode:**
+  - Click on the icon with the letter **"R"** to enter phase correction mode for the rows, which correspond to the F2 dimension.
+
+5. **Set the Pivot Point:**
+  - Right-click at the left limit of the screen and select **"Set Pivot"**.
+  - Place the pivot point away from the peaks to avoid hiding the frequencies we want to phase.
+  - Note: Only the `PHC0` value will be adjusted, not `PHC1`, so a pivot line is not necessary.
+
+![F2 Before Phasing](images/F2_before_phasing.png)
+
+6. **Adjust the Phase:**
+   - Left-button press on the **"0"** icon and drag your mouse until the strongest signals appear at the top part of each panel.
+   - Ensure the frequency lines from end to end look straight and balanced.
+
+![F2 After Phasing](images/F2_after_phasing.png)
+
+7. **Save the PHC0 Value:**
+   - Click on the **"Save-disk"** icon to save the selected `PHC0` value (e.g., `-115` degrees) to the N-HN plane spectrum.
+   - Then click the **Return icon**.
+
+8. **Verify the Phase-Corrected Spectrum:**
+   - The phase-corrected spectrum should now appear without any anti-phase peak components.
+
+![Phased 34 Plane](images/phased_34_plane.png)
+
+9. **Save and Apply PHC0:**
+  - Note down the optimal `PHC0` value (e.g., `-115.0` degrees) and click the **Save-disk nD** icon to copy the values 
+to the 2D plane. Then click the **Return icon**.
+  - Transfer the `PHC0` value to the raw (unprocessed) 4D spectrum, using the `PHC1` value of `0`.
```

Comment:
I don't understand the zero-ing the PHC1 value. In principle, just the PHC0 correction should suffice. But if there has to be PHC1 too, then transferring only the PHC0 will not result in the same phasing of the 4D as the 2D. So it is just enough to say: "Transfer the phasing coefficients PHC0 and, if used, PHC1 to the raw 4D spectrum"